.github/workflows/main.yml

```diff
@@ -2,40 +2,297 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+# TODO: restore after testing
+#    branches:
+#      - master
+#  pull_request:
+#    branches:
+#      - master
+
+env:
+ # TODO: remove after testing
+  DIST_ROOT: '/ipfs/QmX6J3hVtMF9Y73CKcyLHfgviEfXGFFQABWPXsD17EsBhg' # dist.ipfs.io without ipfs-update@v1.7.1
```

Comment:
This is so we run build against  CID that represents dist.ipfs.io without ipfs-update@v1.7.1 – this demonstrates how additional entry in `dists/ipfs-update/versions`  triggers build of "missing" binary artifacts.

(we will remove this before merging this PR)